glusterd2/bricksplanner/planner.go

```diff
@@ -81,10 +81,6 @@ func getBricksLayout(req *api.VolCreateReq) ([]api.SubvolReq, error) {
 		subvolSize = subvolSize / uint64(numSubvols)
 	}
 
-	if req.SnapshotReserveFactor < 1 {
```

Comment:
why this is removed?

Reply:
The same check was made before calling this function. so this was redundant. If you would like to keep this check in this function then I can remove the check made in the calling function.